components/maskedtextbox/appearance.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
In the other components documentation the Appearance article is placed last, lets keep that manner for all of them